dashboard_desc.txt

```diff
@@ -0,0 +1,6 @@
+For building the dashboard, I liked the mockup given and decided to build the project off it. I wanted a minimalist design and so I stuck with a two-tone background, I choose a light green, in homage to Sidekick's green tone.
+
+I used bootstrap 4.0 for their powerful flexbox layout. Each course is it's own row and the credit squares are nested flexboxes.
```

Comment:
If you didn't have a grid-based css framework like Bootstrap, how else could you accomplish this layout?

Reply:
From what I know CSS has flexbox support so that was what I was leaning towards. However while googling I also found CSS Grid which looks to be more powerful that flexbox (is 2-dimensional). So I would go with CSS Grid.